lib/config.rb

```diff
@@ -23,7 +24,8 @@ module Config
     knockout_prefix: nil,
     merge_nil_values: true,
     overwrite_arrays: true,
-    merge_hash_arrays: false
+    merge_hash_arrays: false,
+    validation_contract: nil
```

Comment:
I am still wondering if we should call it `validation_contract` or simply `contract`. @pyromaniac @rdubya @supremebeing7

Reply:
I'm in favor of being explicit - `validation_contract`